src/app/modules/main/chat_section/module.nim

```diff
@@ -108,14 +108,19 @@ proc buildChatUI(self: Module, events: EventEmitter,
     var chatImage = c.identicon
     var isIdenticon = false
     var isUsersListAvailable = true
+    var blocked = false
     if(c.chatType == ChatType.OneToOne):
       isUsersListAvailable = false
-      (chatName, chatImage, isIdenticon) = self.controller.getOneToOneChatNameAndImage(c.id)
+      let contactDetails = self.controller.getContactDetails(c.id)
+      chatName = contactDetails.displayName
+      chatImage = contactDetails.icon
+      isIdenticon = contactDetails.isIdenticon
+      blocked = contactDetails.details.isBlocked()
 
     let amIChatAdmin = self.amIMarkedAsAdminUser(c.members)
 
     let item = initItem(c.id, chatName, chatImage, isIdenticon, c.color, c.description, c.chatType.int, amIChatAdmin, 
-    hasNotification, notificationsCount, c.muted, active = false, c.position, c.categoryId)
+    hasNotification, notificationsCount, c.muted, blocked=blocked, active=false, c.position, c.categoryId)
```

Comment:
Should it be only passing `blocked`without the need of assignement?

Reply:
@noeliaSD please check few lines above, `blocked` is the variable defined locally for that proc and `blocked` is the name of the parameter expected in `initItem`.

Reply:
> @noeliaSD please check few lines above, `blocked` is the variable defined locally for that proc and `blocked` is the name of the parameter expected in `initItem`.

Got it! :)

Reply:
It is still correct to say that it's not necessary to define the param name. It could have been only `c.muted, blocked,...`

IMO, defining the param name is only necessary when the value passed is not clear, line the next one. Saying the param is `active` is helpful, because `true` doesn't mean much.

Anyway, all that to say that you can remove `blocked=` here.